addons/hr_expense/models/hr_expense.py

```diff
@@ -471,6 +465,21 @@ def _add_followers(self):
         users = self._get_users_to_subscribe()
         self.message_subscribe_users(user_ids=users.ids)
 
+    @api.model
+    def _update_follower_values_on_create(self, values):
+        # Add the followers at creation, so they can be notified
+        employee_id = values.get('employee_id')
+        if not employee_id:
+            return
+
+        employee = self.env['hr.employee'].browse(employee_id)
+        users = self._get_users_to_subscribe(employee=employee) - self.env.user
+        message_follower_ids_values = []
+        mail_follower_obj = self.env['mail.followers']
```

Comment:
I think that the common convention is to use camel case for models: `MailFollowers`.

Reply:
I agree. Didn't want to change it because there would be a mixed of camel case and non camel case in the same file. I will change it.